guides/storefront-guide.md

```diff
@@ -0,0 +1,821 @@
+## At a glance
+
+Mailchimp Open Commerce (MOC) is a headless commerce platform, which means that its focus is on providing a top-notch server API rather than on UI. But this doesn't mean you're completely on your own.
```

Comment:
Mailchimp Open Commerce is a headless commerce platform, with a focus on providing a server API rather than a user interface. But this doesn't mean you're completely on your own for UI!

Reply:
updated!